src/main/java/CurrenciesExchangeRate.java

```diff
@@ -0,0 +1,36 @@
+public class CurrenciesExchangeRate {
+
+
+
+    public CurrenciesExchangeRate() {
```

Comment:
As mentioned above - constructor should be with 3 parameters, we don't need setters and getters. If you want to print object details use toString method.